File: StockMarketTracker.Client/src/types/Stock.ts
export type Stock = {
	id: string;
	name: string;
	symbol: string;
	exchange: string;
	prevDay: StockPrice;
	priceHistory: StockPrice[];
};

export type StockPrice = {
	low: number;
	open: number;
	date: string;
	high: number;
	close: number;
	volume: number;
};

File: StockMarketTracker.API/StockMarketTracker.Model/Stock.cs
namespace StockMarketTracker.Model
{
    public class Stock
    {
        public Guid Id { get; set; }
        public string Name { get; set; } = string.Empty;
        public string Symbol { get; set; } = string.Empty;
        public string Exchange { get; set; } = string.Empty;
        public IEnumerable<StockPrice> PriceHistory {  get; set; } = Enumerable.Empty<StockPrice>();
    }
}
